pyproject.toml

```diff
@@ -30,12 +33,9 @@ pytest = "^8.4"
 pytest-black = "^0.3.7"
 pytest-cov = "^2.8"
 pytest-flake8 = "^1.0"
-pytest-mypy = "^0.8"
+pytest-mypy = "^1.0"
 pytest-pydocstyle = "^2.3"
-sphinx = "^5.0"
-sphinx-readable-theme = "^1.3.0"
-ghpusher = "^0.2.2"
```

Comment:
Now unused, the sphinx recommended way to push to github pages is using a github actions workflow with a set of actions they recommend.